### PR TITLE
Add "Poll SCM" post-build step

### PR DESCRIPTION
**Pull request by nephre**

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/pollscm/SCMPollPublisher.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    
+    Map<Thread, StackTraceElement[]> map = Thread.getAllStackTraces();
+    for (Thread thread : map.keySet()) {
+      if (thread.getName().contains("SCM polling"))
+      {
+        taskListener.getLogger().println("Contains scm polling.");
+        if (thread.getName().contains(project.getName()))
+        {
+          taskListener.getLogger().println("Interrupting thread.");
+          thread.interrupt();
```
Comment (olivergondza) [inline src/main/java/org/jenkinsci/plugins/pollscm/SCMPollPublisher.java]: This is not a sufficient guarantee you are killing the right thread. Dangerous.

Also, when the trigger is already running, what is the point to kill it and start it again?

**Comment by galbramc** [comment]
This is a feature I would really like to have. Any chance it could get cleaned up and released? If I knew Java I would help...

**Comment by nephre** [comment]
> This is a feature I would really like to have. Any chance it could get cleaned up and released? If I knew Java I would help...

I forgot about it, sorry :) I'll have to go back to this and review again

**Comment by galbramc** [comment]
Awesome! I would greatly appreciate that. Let me know if I can help with testing it out.

**Comment by MarkEWaite** [comment]
Closing pull request as no progress.  Can be reopened if there is enough interest and someone that makes the necessary changes.

**Pull request closed without merging**